app/javascript/controllers/turbo_search_controller.js

```diff
@@ -0,0 +1,8 @@
+import { Controller } from "@hotwired/stimulus"
+
+// Connects to data-controller="turbo-search"
+export default class extends Controller {
+  search(){
+    this.element.requestSubmit()
```

Comment:
This works, but since we're not doing the filter-as-you-type thing, you should be using [`pushState`](https://developer.mozilla.org/en-US/docs/Web/API/History/pushState) here so that your back button works as expected.